doc/nrf/releases_and_maturity/migration/migration_guide_2.4.99-cs3_to_2.6.99-cs2.rst

```diff
@@ -0,0 +1,20 @@
+.. _migration_cs3_to_2_6_99_cs2:
+
+Migration notes for |NCS| v2.6.99_cs2 for v2.4.99-cs3 users
+###########################################################
+
+This document describes the new features and changes implemented in |NCS| to take into consideration when migrating your application from |NCS| for the nRF54 customer sampling release v2.4.99-cs3 to |NCS| v2.6.99-cs2.
```

Comment:
```suggestion
This document describes the new features and changes implemented in |NCS| to take into consideration when migrating your application from |NCS| for the nRF54H customer sampling release v2.4.99-cs3 to |NCS| v2.6.99-cs2.
``` 
?

Reply:
No, that release was for both 54H and 54L, so 54 is actually correct.